Cargo.toml

```diff
@@ -1,23 +1,19 @@
 [package]
-name = "cargo"
```

Comment:
You seem to have a tool or something that automatically reformatted this file and reordered everything. It makes the diff harder to understand because at face value it looks like you made changes you didn't make. Could you avoid making cosmetic changes to code you don't need to change for this PR?